README.mdown

```diff
@@ -32,3 +32,20 @@ To run tests do:
     python2 bookshelf/api_v2/test_ec2.py
     python2 bookshelf/api_v2/test_rackspace.py
 
+There are also tests for the api_v3 that spin up GCE, Rackspace, and EC2
+instances. These tests require credentials to access GCE, Rackspace, and EC2.
+In an effort to attempt to reduce the amount of work to set up the
+configuration, they build on the existing acceptance.yml as described at:
+http://doc-dev.clusterhq.com/gettinginvolved/appendix.html#acceptance-testing-configuration.
+
```

Comment:
Removed. I still call it `acceptance.yml`, and right now the configuration is compatible with a ClusterHQ flocker acceptance.yml (although it is extended to have ssh_keys). Personally I'm going to use the same file on my machine. However, with this sentence removed I no longer think we have any requirement to preserve that compatibility long-term.